mavros_msgs/msg/ADSBVehicle.msg

```diff
@@ -0,0 +1,17 @@
+# The location and information of an ADSB vehicle
+#
+# https://pixhawk.ethz.ch/mavlink/#ADSB_VEHICLE
+
+std_msgs/Header header
+
+uint32 ICAO_address
+geographic_msgs/GeoPoint lla
+string altitude_type # Type from ADSB_ALTITUDE_TYPE enum
```

Comment:
you want me to add 29 constants to a msg just because you don't like a `to_string` parser on the cb?